File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/VenueFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.Venue;

public class VenueFilterValidator extends FilterValidator<Venue> {

    public VenueFilterValidator(CustomValidator<Venue> validator) {
        super(Venue.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/VenueFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.telegram.bot.api.model.objects.std.Venue
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class VenueFilterValidator(validator: Validator<Venue>) : FilterValidator<Venue>(Venue::class, validator)

